app/views/layouts/application.html.erb

```diff
@@ -46,7 +45,7 @@
         <!-- Add styling to center -->
         <p>Made with love by the #April2017 <a class="footer-link" href="http://makersacademy.com">Makers Academy</a> cohort</p>
       </div>
-
+      <%= javascript_include_tag 'application' %>
     </footer>
```

Comment:
It might be dangerous to change how the JS is required in rails, you should try using document.ready.